sphinx/tutorial/rose/suites.rst

```diff
@@ -395,78 +402,85 @@ In this example the ``hello`` task will run the application located in
    default=echo "Hello World!"
 
 The name of the application to run can be overridden using the ``--app-key``
-command line option or the :envvar:`ROSE_TASK_APP` environment variable. For
+command-line option or the :envvar:`ROSE_TASK_APP` environment variable. For
 example the ``greetings`` :term:`task` will run the ``hello`` :term:`app <Rose
 app>` in the task defined below.
 
 .. code-block:: cylc
    :caption: suite.rc
 
    [runtime]
-      [[greetings]]
-         script = rose task-run --app-key hello
+       [[greetings]]
+           script = rose task-run --app-key hello
 
 
 Rose Bush
 ---------
 
 .. TODO - move this into the Cylc tutorial with the upcomming Cylc "nameless".
 
-Rose provides a utility for viewing the status and logs for Cylc suites called
-Rose Bush. Rose Bush displays suite information in web-pages.
+Rose provides a utility for viewing the status and logs of Cylc suites called
+Rose Bush. Rose Bush displays suite information in web pages.
 
 .. figure:: img/rose-bush.png
    :alt: Rose Bush screenshot
    :width: 100%
 
    Screenshot of a Rose Bush web page.
 
-If a Bose Bush server is provided at your site you can open the Rose Bush
+If a Rose Bush server is provided at your site, you can open the Rose Bush
 page for a suite by running the :ref:`command-rose-suite-log` command
 in the suite directory.
 
-Otherwise an add-hoc web-server can be set up using the
-:ref:`command-rose-bush` ``start`` command.
+Otherwise an add-hoc web server can be set up using the
+:ref:`command-rose-bush` ``start`` command argument.
 
 
 .. practical::
 
    .. rubric:: In this practical we will take the ``forecast`` Rose application
       that we developed in the :ref:`Metadata Tutorial <tutorial-rose-metadata>`
-      and integrate it into the :ref:`Weather Forecasting Suite
+      and integrate it into the :ref:`weather-forecasting suite
       <tutorial-datetime-cycling-practical>`.
 
-   Move into the suite directory from the previous practical::
+   #. **Set-up The Forecast Application.**
```

Comment:
In other practicals trivial setup like moving into a particular directory has been done before the first item. As the only instruction in here is to `cd` into a directory I would leave this as is.

Reply:
I agree! I had actually introduced this when I thought I had solved the (not even broken) tutorial by adding back in the ``RESOLUTION`` and ``DOMAIN`` settings in the ``[runtime][root][environment]`` section, my head being a bit frazzled as the whole point of the tutorial was not to do this, as I realised 5 minutes later. I should have reverted this change along with that, but missed it. My apologies.

Reply:
No probs.